runtime/eventlog.c

```diff
@@ -41,32 +41,80 @@
t/gc_stat\",\n    \"explicit/gc_minor\",\n    \"explicit/gc_major\",\n    \"explicit/gc_full_major\",\n    \"explicit/gc_compact\",\n    \"major\",\n    \"major/roots\",\n    \"major/sweep\",\n    \"major/mark/roots\",\n    \"major/mark/main\",\n    \"major/mark/final\",\n    \"major/mark\",\n    \"major/mark/global_roots_slice\",\n    \"major_roots/global\",\n    \"major_roots/dynamic_global\",\n    \"major_roots/local\",\n    \"major_roots/C\",\n    \"major_roots/finalised\",\n    \"major_roots/memprof\",\n    \"major_roots/hook\",\n    \"major/check_and_compact\",\n    \"minor\",\n    \"minor/local_roots\",\n    \"minor/ref_tables\",\n    \"minor/copy\",\n    \"minor/update_weak\",\n    \"minor/finalized\",\n    \"explicit/gc_major_slice\"\n};\n\n/*\n\nMiscellaneous GC counters\n\n*/\nenum gc_counter : uint16_t {\n    \"alloc_jump\",\n    \"force_minor/alloc_small\",\n    \"force_minor/make_vect\",\n    \"force_minor/set_minor_heap_size\",\n    \"force_minor/weak\",\n    \"force_minor/memprof\",\n    \"major/mark/slice/remain\",\n    \"major/mark/slice/fields\",\n    \"major/mark/slice/pointers\",\n    \"major/work/extra\",\n    \"major/work/mark\",\n    \"major/work/sweep\",\n    \"minor/promoted\",\n    \"request_major/alloc_shr\",\n    \"request_major/adjust_gc_speed\",\n    \"request_minor/realloc_ref_table\",\n    \"request_minor/realloc_ephe_ref_table\",\n    \"request_minor/realloc_custom_table\"\n};\n\n/*\n\nBlock allocation counters, per size buckets.\n\n*/\nenum alloc_bucket : uint8_t {\n  \"alloc 01\" = 1,\n  \"alloc 02\",\n  \"alloc 03\",\n  \"alloc 04\",\n  \"alloc 05\",\n  \"alloc 06\",\n  \"alloc 07\",\n  \"alloc 08\",\n  \"alloc 09\",\n  \"alloc 10-19\",\n  \"alloc 20-29\",\n  \"alloc 30-39\",\n  \"alloc 40-49\",\n  \"alloc 50-59\",\n  \"alloc 60-69\",\n  \"alloc 70-79\",\n  \"alloc 80-89\",\n  \"alloc 90-99\",\n  \"alloc large\"\n};\n\n/*\n\nEach event is comprised of the previously defined event.header\nand the fields defined here.\n\nAn entry event marks the start of a gc phase.\n\n*/\nevent {\n    id = 0;\n    name = \"entry\";\n    stream_id = 0;\n    fields := struct {\n        enum gc_phase phase;\n    };\n};\n\n/*\n\nexit counterparts to entry events\n\n*/\nevent {\n    id = 1;\n    name = \"exit\";\n    stream_id = 0;\n    fields := struct {\n        enum gc_phase phase;\n    };\n};\n\nevent {\n    id = 2;\n    name = \"counter\";\n    stream_id = 0;\n    fields := struct {\n        uint64_t count;\n        enum gc_counter kind;\n    };\n};\n\nevent {\n    id = 3;\n    name = \"alloc\";\n    stream_id = 0;\n    fields := struct {\n        uint64_t count;\n        enum alloc_bucket bucket;\n    };\n};\n\n/*\n Flush events are used to track the time spent by the tracing runtime flushing\n data to disk, useful to remove flushing overhead for other runtime mesurements\n in the trace.\n*/\nevent {\n     id = 4;\n     name = \"flush\";\n     stream_id = 0;\n     fields := struct {\n        uint64_t duration;\n     };\n};\n";
```

Comment:
You can break constant strings into multiple lines as:

```c
char *my_string = "Line 1 "
                  "Line 2";
```

which is the same as 

```c
char *my_string = "Line 1 Line 2";
```